Neos.Http.Factories/Classes/HttpObjectFactory.php

```diff
@@ -0,0 +1,21 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\Http\Factories;
+
+use Psr\Http\Message\RequestFactoryInterface;
+use Psr\Http\Message\ResponseFactoryInterface;
+use Psr\Http\Message\ServerRequestFactoryInterface;
+use Psr\Http\Message\StreamFactoryInterface;
+use Psr\Http\Message\UploadedFileFactoryInterface;
+use Psr\Http\Message\UriFactoryInterface;
+
+class HttpObjectFactory implements ServerRequestFactoryInterface, RequestFactoryInterface, ResponseFactoryInterface, UriFactoryInterface, StreamFactoryInterface, UploadedFileFactoryInterface
```

Comment:
One possible alternative name for this class could be PsrHttpFactory, as it clearly indicates that it implements the PSR HTTP message factories. Another option could be HttpMessageFactory, as it emphasizes the fact that this class is responsible for creating HTTP messages.